src/test/java/hudson/tasks/junit/SuiteResultTest.java

```diff
@@ -38,6 +38,10 @@
 
 import junit.framework.TestCase;
 
+import static org.hamcrest.core.StringStartsWith.startsWith;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
 /**
```

Comment:
I modified it slightly and amended the commit, to see if it works. The problem is that I don't get the error locally, only on CI.